python/cudf/cudf/_lib/binaryop.pyx

```diff
@@ -212,11 +212,7 @@ def binaryop(lhs, rhs, op, dtype):
             c_op,
             c_dtype
         )
-
-    if is_string_col is True:
-        return handle_null_for_string_column(result, op.name.lower())
```

Comment:
🔥deleting code is very cool. Is `handle_null_for_string_column` now unused? Can we delete that too?

Reply:
Gone!